core/variables.js

```diff
@@ -299,6 +299,7 @@ Blockly.Variables.createVariable = function(workspace, opt_callback, opt_type) {
   // Prompt the user to enter a name for the variable
   Blockly.prompt(newMsg, '',
       function(text, additionalVars, variableOptions) {
+        variableOptions = variableOptions || {};
```

Comment:
Thanks for fixing this! I probably didn't test the playground when making changes here for cloud variables.